rent.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
```

Comment:
- [ ] all three variables should have reader and writer access kindly fix this by adding the person and book to attr_accessor instead